src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -71,7 +71,7 @@ const CreateConceptForm = (props) => {
           <div className="form-group col-md-5">
             <label htmlFor="class"><h5>Class</h5></label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding')
```

Comment:
Could we make these constants so that we don't leave room for having this issue in the future?

Reply:
Done